### ISSUE TITLE
Validation

### ISSUE THREAD

**Issue by caiyongqi** [CCIIPLab/GCE-GNN#16]
Excuse me. Does the model that get the results recorded in the paper use the validation set for training? Are the hyperparameters obtained on the validation set first, and then the model is retrained on the training set and validation set? I can't seem to get the results in the paper.
